src/camvidlog2/yoloe/cli.py

```
@@ -59,5 +61,30 @@ def load(
         print("saved file")
 
 
+@app.command()
+def show(
```

Ruff (0.11.9)

Missing return type annotation for public function `show`

Add return type annotation: `None`

(ANN201)